pages/common/virsh.md

```diff
@@ -8,13 +8,13 @@
 
 `virsh connect {{qemu:///system}}`
 
-- List all domains:
+- Activate a network named `default`:
 
-`virsh list --all`
+`virsh net-start {{default}}`
```

Comment:
```suggestion
`virsh net-start default`
```

Reply:
The device name can change if you've created more than one.

Reply:
The example description states that the name is "default," and we do not use placeholders in these cases.

Reply:
I've done that multiple times in the past. This is the first time anyone's taken note of it.

Reply:
I also think that it would be clearer to have a placeholder in this case.

Reply:
There were some PRs removing placeholders in similar cases, for example, https://github.com/tldr-pages/tldr/pull/13723

Reply:
I really disagree with these changes. For me placeholders are there to indicate that an argument can be changed.

Reply:
I don't have a strong opinion either way, so it is fine for me to include the placeholder here.